src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -1855,11 +1855,16 @@ class A { }");
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsSimplifyTypeNames)]
         public void TestMissingNullableSimplificationInsideCref4()
         {
-            TestMissing(
+            Test(
 @"using System;
```

Comment:
Yes it should. Will fix.

    -- Cyrus
________________________________
From: Dustin Campbell<mailto:notifications@github.com>
Sent: ‎9/‎12/‎2015 7:11 AM
To: dotnet/roslyn<mailto:roslyn@noreply.github.com>
Cc: Cyrus Najmabadi<mailto:cyrusn@microsoft.com>
Subject: Re: [roslyn] Share test code (#5179)


In src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn%2fpull%2f5179%23discussion_r39337623&data=01%7c01%7ccyrusn%40microsoft.com%7c051a859c27374ef9aa6a08d2bb7c0558%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=3aBMsgpluDLMGkPdQSSx4TnMwUPa1iRbDDCLJ7A3K8E%3d>:

> @@ -1855,11 +1855,16 @@ class A { }");
>          [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsSimplifyTypeNames)]
>          public void TestMissingNullableSimplificationInsideCref4()
>          {
> -            TestMissing(
> +            Test(
>  @"using System;


Should the name of the test be different then?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn%2fpull%2f5179%2ffiles%23r39337623&data=01%7c01%7ccyrusn%40microsoft.com%7c051a859c27374ef9aa6a08d2bb7c0558%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=h5lKe1ngCPhqv8zCksLEHaUVcT7kSVoAcLF9Zr0x6tE%3d>.